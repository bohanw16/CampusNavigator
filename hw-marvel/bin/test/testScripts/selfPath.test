# Find a path from a node to itself using selfPath.csv

LoadGraph selfPath selfPath.csv

FindPath selfPath n1 n1