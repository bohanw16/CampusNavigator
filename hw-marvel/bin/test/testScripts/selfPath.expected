# Find a path from a node to itself using selfPath.csv

loaded graph selfPath

path from n1 to n1: